src/test/java/il/ac/technion/cs/fling/examples/usecases/Json.java

```diff
@@ -0,0 +1,44 @@
+package il.ac.technion.cs.fling.examples.usecases;
+
+import static il.ac.technion.cs.fling.examples.generated.Json.value;
+import static il.ac.technion.cs.fling.examples.generated.Json.array;
+import static il.ac.technion.cs.fling.examples.generated.Json.object;
+
+// @formatter:off
+public class Json {
+  public static void main(String[] args) {
+    value("f");
```

Comment:
If you want, you can practice AST traversal by, say, pretty-printing the JSON objects. You can check the other use-cases for examples.